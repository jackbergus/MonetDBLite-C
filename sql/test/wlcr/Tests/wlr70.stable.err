stderr of test 'wlr70` in directory 'sql/test/wlcr` itself:


# 21:25:59 >  
# 21:25:59 >  "/usr/bin/python2" "wlr70.py" "wlr70"
# 21:25:59 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/wlcr//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	monet_prompt = 
# cmdline opt 	mapi_port = 59537
# cmdline opt 	mapi_usock = /var/tmp/mtest-30891/.s.monetdb.59537
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone
# cmdline opt 	gdk_debug = 536870922

# 21:26:01 >  
# 21:26:01 >  "Done."
# 21:26:01 >  

