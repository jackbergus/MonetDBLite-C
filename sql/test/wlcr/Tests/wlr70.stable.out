stdout of test 'wlr70` in directory 'sql/test/wlcr` itself:


# 21:25:59 >  
# 21:25:59 >  "/usr/bin/python2" "wlr70.py" "wlr70"
# 21:25:59 >  

# MonetDB 5 server v11.26.0
# This is an unreleased version
# Serving database 'mTests_sql_test_wlcr-clone', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:59537/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-30891/.s.monetdb.59537
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
#sleep 1000 ms
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000025
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000026
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000027
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000028
#sleep 1000 ms
#select * from tmp;
% sys.tmp,	sys.tmp # table_name
% i,	s # name
% int,	clob # type
% 2,	15 # length
[ 35,	"resumed logging"	]
#call waitformaster();
#waiting for master 25 < 29
#in sync with master 29 == 29
#select * from tmp;
% sys.tmp,	sys.tmp # table_name
% i,	s # name
% int,	clob # type
% 3,	31 # length
[ 35,	"resumed logging"	]
[ 100,	"inserted while logging was idle"	]

# 21:26:01 >  
# 21:26:01 >  "Done."
# 21:26:01 >  

